app/src/main/res/values/dimens.xml

```diff
@@ -3,4 +3,7 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+
+    <dimen name="tags_padding_vertical">5</dimen>
```

Comment:
I think you should put the metric for dimen, if you want to only have integer, you can use the integers.xml file